### ISSUE TITLE
question: Can I use static tokens?

### ISSUE THREAD

**Issue by nirajd** [greenpau/caddy-auth-portal#213]
for certain parts of my environment, I want to use a static token (something akin to api keys for github etc). Is there a way to do that? I would like to authenticate once, copy the generated token or key and embed that in other appilications that can authenticate perpetually based on that key.

I  read a bunch but couldn't figure that out.

**Comment by greenpau** [comment]
@nirajd, I am working on a feature to allow generating API keys for a specific user. Stay tuned.

**Comment by greenpau** [comment]
@nirajd , this feature is now available in [v1.4.37](https://github.com/greenpau/caddy-auth-portal/releases/tag/v1.4.37).

Please see https://gist.github.com/greenpau/6266cebb90c75614e90b1e5c44d339ef

Note the `with api key auth realm local`.

Additionally, you need to "Add API Key" in "Settings":

![image](https://user-images.githubusercontent.com/3826416/148348288-2bbe0271-1a80-4a12-9e1c-c86a0685c0f8.png)

![image](https://user-images.githubusercontent.com/3826416/148348208-1473d5ae-f5b0-4bc4-9975-1c5a5014e5c6.png)


Once configured, you should be able to access endpoints with API keys:

```bash
curl -v -H 'X-Api-Key: AnpBUzrkuCSYrmq7H5RoIr9xdmtqBxe8Joo1V3p8exGmYC6TYVGeyicCo1VKLsmABNNqibA2mCb28OU7Z7Wx' https://assetq.myfiosgateway.com:8443/
```

**Comment by nirajd** [comment]
Great! Thanks.
